src/fontra/client/web-components/plugin-manager.js

```diff
@@ -50,18 +51,30 @@ export class PluginManager extends SimpleElement {
   }
 
   async promptAddPlugin(text = "") {
-    const newPluginPrompt = await dialogSetup("Add plugin", "", [
-      { title: "Cancel", resultValue: "no", isCancelButton: true },
-      { title: "Create", resultValue: "ok", isDefaultButton: true, disabled: true },
-    ]);
+    const newPluginPrompt = await dialogSetup(
+      translate("plugin-manager.dialog.title"),
```

Comment:
Suggest to use a less generic key prefix for these. What about `plugin-manager.add-plugin-dialog.xxx`?

Reply:
thanks. done.